go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/BurntSushi/toml v0.4.1
 	github.com/Xuanwo/go-bufferpool v0.2.0
 	github.com/docker/go-units v0.4.0
+	github.com/gobwas/glob v0.2.4-0.20181002190808-e7a84e9525fe
```

Comment:
Please always use tagged version unless we don't have choice.

Reply:
Got it.